analytics/tracker.go

```diff
@@ -128,9 +129,6 @@ func NewDefaultTracker() Tracker {
 
 	logger := log.NewUtilsLogAdapter()
 	tracker := analytics.NewDefaultTracker(&logger)
-	if stateChecker.UseAsync() {
-		tracker = analytics.NewDefaultTracker(&logger)
-	}
```

Comment:
I did this dumb thing back in https://github.com/bitrise-io/bitrise/commit/08f5cd13a0956474264a423b576e06be2f3735f6#diff-ee3fd0c2a79d62b7da48113df24fa0c13b0d3e8db9cdc7557433f4ae63c13115R123